packages/logger/package.json

```diff
@@ -16,19 +16,35 @@
     "test:e2e:nodejs18x": "RUNTIME=nodejs18x jest --group=e2e",
     "test:e2e": "jest --group=e2e",
     "watch": "jest --watch --group=unit",
-    "build": "tsc --build --force",
+    "build:cjs": "tsc --build --force",
+    "build:esm": "tsc --project tsconfig.esm.json",
+    "build": "npm run build:esm & npm run build:cjs",
     "lint": "eslint --ext .ts,.js --no-error-on-unmatched-pattern .",
     "lint-fix": "eslint --fix --ext .ts,.js --no-error-on-unmatched-pattern .",
     "prebuild": "rimraf ./lib",
+    "postbuild": "node ./scripts/buildExportPackageJson.js",
     "prepack": "node ../../.github/scripts/release_patch_package_json.js ."
```

Comment:
can we add this ` && rimraf ./lib/tsconfig.*.tsbuildinfo` so we remove the incremental build files from the packed output?

Reply:
Added `&& rimraf ./lib/*.tsbuildinfo` to remove the incremental build files